### PR TITLE
Korean translation of Platforms paper

### PR DESCRIPTION
**Pull request by sysnet4admin**
Would you merge for korea version for index.md?

### CODE REVIEW

**Comment by joshgav** [comment]
Thanks for this @sysnet4admin! Two things we need to do:

- Make sure we've set up the directory properly for Hugo to handle multiple languages, along the lines of https://github.com/cncf/tag-app-delivery/issues/394
- Check the translated version here with Google Translate for consistency

I'll work on the second one now and the first soon.

**Comment by joshgav** [comment]
Here's a link that includes a Google Translate version for others that want to review: <https://github-com.translate.goog/cncf/tag-app-delivery/blob/2a1002a23187fa203525bcb81b2ad05d5956a72a/platforms-whitepaper/v1/index-ko.md?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp>

File: platforms-whitepaper/v1/index-ko.md
Comment (joshgav) [inline platforms-whitepaper/v1/index-ko.md:1]: Per [these docs at the Hugo site](https://gohugo.io/content-management/multilingual/#translation-by-filename) this file needs to be named `index.ko.md` to be linked to `index.md`. Can you change that?

**Review by joshgav — CHANGES_REQUESTED** [review]
The text looks pretty good to me based on applying Google Translate! Can you change the name of the file per Hugo's guidelines and my comment?

@cjyabraham can you add `platforms-whitepaper/v1` to the paths which trigger a rebuild in Netlify? Thanks!

File: platforms-whitepaper/v1/index-ko.md
Comment (cjyabraham) [reply platforms-whitepaper/v1/index-ko.md:1]: I don't know how to get netlify to watch a file outside of the `website` directory. Would you consider we restructure the whole repo so the website resides in the root and the whole repo can then be watched?

Comment (sysnet4admin) [reply platforms-whitepaper/v1/index-ko.md:1]: Okay @joshgav. 
So may I PR again to change file name from `index-ko.md` to `index.ko.md` now? 
Because the structure of website directory is not clear yet.
Feel free to let me know how do I do.

Comment (joshgav) [reply platforms-whitepaper/v1/index-ko.md:1]: Hi @sysnet4admin, yes please name the file `index.ko.md`, you can add a commit to this PR if you'd like.

Then @cjyabraham let's force a redeploy for this PR and check out the results (we'll figure out the triggering issue somewhere else).

Thank you!

Comment (sysnet4admin) [reply platforms-whitepaper/v1/index-ko.md:1]: @joshgav 
Changed file name from `index-ko.md` to `index.ko.md` . 
Feel free to let you know further activity if it needs.

**Comment by joshgav** [comment]
@cjyabraham could you force the preview to deploy for this PR?

Maybe we should just have it deploy for every PR for now, even outside the `website` dir.

**Comment by cjyabraham** [comment]
> @cjyabraham could you force the preview to deploy for this PR?
> 
> Maybe we should just have it deploy for every PR for now, even outside the `website` dir.

Ok, here's the preview: https://deploy-preview-406--tag-app-delivery.netlify.app/
And I think Netlify should now be looking for changes in the whitepapers directories.

**Comment by joshgav** [comment]
Hi @sysnet4admin - following #492 the website supports multiple languages, and so - sorry to have to ask :laughing: - but could we refactor this PR to be like #492 and have Korean as an additional supported language on the site?

cc @eeap who volunteered to help in <https://github.com/cncf/tag-app-delivery/issues/394#issuecomment-1827680971>

**Comment by sysnet4admin** [comment]
@joshgav Sure! You may know that I would like to spread out useful information like this. Feel free to move forward!.

**Comment by sysnet4admin** [comment]
@Eeap No problem totally. Any help or take over will highly welcome :) 
Please go for us

**Comment by abangser** [comment]
Thanks for sticking with this @sysnet4admin !

I went to test this and I don't think the file is in the right location for the current hugo setup. I think if you instead move the file to replace this one the one at `website/content/ko/wgs/platforms/whitepaper/index.md` we will be able to get the translation on the site.

In addition, I saw that @Eeap is up for helping. @Eeap based on your hope to use this I gather you may also be a native Korean speaker? We usually try to have at least one other native speaker review the translations as we know that Google Translate isn't always the best at picking up nuance of real language use. Is that something you could help with?

I think those two bits are parallelisable and hopefully the last hurdles to getting this merged! 🤞

**Comment by sysnet4admin** [comment]
Hi @abangser 
Thank you for your feedback. 

**1. index.ko.md**
First I already PR #406 before for index.ko.md. probably it would like to merge this `md` from current index.md right?
If it is Yes, I will progress from #406 during weekend. But I am wrong, feel free to let me know. 

As-IS: not merged yet (#406)
To-BE: website/content/ko/wgs/platforms/whitepaper/index.md

**2. localization** 
It looks like double-check by native-speaker when it uploaded or released? from other native-speaker. 
Yes I could be (if that's not a lot of burden at a once 😅)
@Eeap Any help as current thread?
OR
`parallelisable` may have issue? if it is right, I do just merge only index.ko.md for platforms-whitepaper(v1). 
So maybe @Eeap will lead other docs I guess. Is it right? @Eeap 

Anyway please don't be any frustrating to point out me.

**Comment by Eeap** [comment]
Hi @abangser @sysnet4admin 

If you don't mind I think I can do other contents too. :) But I will be able to translate it maybe next weekend.
Also, I don't know exactly that I understood other things. 😅
Do you mean that if I translate other contents, I have to confirm from just one native korean speaker like @sysnet4admin ?? or more than one ??
Thanks you for suggesting this opportunity.

**Comment by sysnet4admin** [comment]
@abangser 
I replaced `index.md` for platforms-whitepaper as you request. 
Please let me know any issue or further thing, feel free to let me know.  

@Eeap I merged your draft version which is changed the content's title. If you need to change from this translating version (from 8months ago), please let me know or PR from this version. 

FYR @joshgav  :)

**Comment by abangser** [comment]
That's perfect @Eeap! So sorry for the delayed response but the change you made was exactly what I was suggesting and I have confirmed on a local build that I can now see the Korean version as expected.

Thanks for all your hard work on this!

I did also notice the update to the netlify.toml file. I am wondering if @joshgav or maybe @cjyabraham have knowledge on this and can confirm it is a setting we want to apply?

**Comment by cjyabraham** [comment]
Hi @abangser, I've reverted the change to netlify.toml. It was done back last May and we've had more recent updates to it since then.

**Comment by Eeap** [comment]
I think there is something to change about translation style. Because I used "~한다와 같은 평어체." in other contents for following common translation guide, It looks nice to change like this. @sysnet4admin
Also, Glossary page has been changed to another page.

File: website/content/ko/wgs/platforms/whitepaper/index.md
Comment (Eeap) [inline website/content/ko/wgs/platforms/whitepaper/index.md:196]: There is **Messaging** sentence in English page, I think you missed it. :)
If you don't mind, I will fix all of them like style, missing sentence. (And I will make a PR to your branch)

**Review by Eeap — CHANGES_REQUESTED** [review]

**Comment by sysnet4admin** [comment]
@Eeap Thank you for notification. 
So I changed (to split each of purpose). `whitepaper/index.md` to `whitepaper/index.md` and `glossary/_index.md`
However I looks not latest version especially `glossary/_index.md`. Would you update to `latest`? to catch up the glossary?

**Comment by abangser** [comment]
Hi all, thanks again for all the hard work on this translation. 

I just wanted to call out that the glossary has a PR open now for a facelift so if you are interested in updating you may want to refer to the options in [this comment](https://github.com/cncf/tag-app-delivery/pull/545#issuecomment-1925309424)!

**Comment by sysnet4admin** [comment]
@Eeap Hi, 
I already hand over my documents. so you could modify whatever you want. Anyhow when I translate to some document, my point is ONLY to understand easily to user and community. Probably that's main reason why casual describing and more comment by my opinion (like extra information) 
Plus, glossary @abangser said it change a lot (i.e. facelift) you(@Eeap ) may follow up what you want. 
In my view, 2nd option is better but you may choose your own thought.

**Comment by Eeap** [comment]
@sysnet4admin Hi :)
I understand what you said, so I did not touch translation style like extra information. I just have modified sentence closing style, and also created PR to your branch. Could you check it again??

> Plus, glossary @abangser said it change a lot (i.e. facelift) you(@Eeap ) may follow up what you want.
In my view, 2nd option is better but you may choose your own thought.
I have also checked this before, I think it is better to merge this PR(if there is no problem) and create another PR for total Platform WG's translation (It will follow up English contents, and change old version)

**Comment by sysnet4admin** [comment]
@Eeap Thank you for your feedback. 
In my option, first merge and change or modify whatever you want that is better progress I think. 
I am not sure what you metion the `brance` but I do not modify anymore after publishing it. 
So you could take over after finishing this stage. i.e. you could chnage any kind of format or change type.

**Comment by Eeap** [comment]
Okay, it's great :) Thanks for suggesting other options.
Then could you double check if I translate the contents, @sysnet4admin ?? 
Could you check this again, @abangser ?? It is better to create another PR for total Platform WGs translation.

**Comment by sysnet4admin** [comment]
@Eeap I am not sure that you point me out. 
Anyhow here is my suggestion for understading each other. 
`Season1`: I finished this and merge(if it could). 
`Season2`: You(@Eeap) make your own episode. Because I am stuck from other job. (Work, Book and others)

**Comment by Eeap** [comment]
Sounds Great :) Thank you @sysnet4admin

**Comment by abangser** [comment]
Hey all. If I follow, it sounds like we want to try to merge this and then extend in another PR. That sounds like a great plan.

I believe we have two blockers for merging:
1. the DCO needs to be signed (I believe this is for you @sysnet4admin)
2. I am unable to build this locally, I worry that we need to rebase main on this branch, but that also didn't work locally so I am not sure if there is a bigger issue
3.  I believe the two keys [here](https://github.com/cncf/tag-app-delivery/pull/406/files#diff-b3ad2cc1c4d0018c3f8006dbe1b084051b1ff732eda4b25fb94f7f5c56df7108R2-R3) need to stay in english (changing this does not improve the issue in the 2nd point)

**Comment by sysnet4admin** [comment]
Hi @abangser, 
I fix key value issue by abnormal work(KO..) as well as several conflict items. 
And I change 2 docs due to it already translate 11 months ago and there is not a lot of change at that moment. 
So in my view, merge it and take over @Eeap from this content. He/She already [agree](https://github.com/cncf/tag-app-delivery/pull/406#issuecomment-1949923798) this.
Thank you for your all of efforts.

File: website/content/ko/wgs/platforms/whitepaper/index.md
Comment (joshgav) [inline website/content/ko/wgs/platforms/whitepaper/index.md:2]: It seems that the key here needs to stay in English, that is it should be "title:"

I say this cause in the current deployment preview no title is rendered for this section in the Korean version.

File: website/content/ko/wgs/platforms/whitepaper/index.md
Comment (joshgav) [inline website/content/ko/wgs/platforms/whitepaper/index.md:4]: This key should also probably staty in English as "version_info"

File: website/content/ko/wgs/platforms/whitepaper/index.md
Comment (joshgav) [inline website/content/ko/wgs/platforms/whitepaper/index.md:5]: Finally, this key should stay in English as "description".

**Review by joshgav — CHANGES_REQUESTED** [review]
From what I can tell the translations look great but the keys for some front matter need to stay in English, as I noted in comments. Let's fix those before we merge, thanks!

File: website/content/ko/wgs/platforms/whitepaper/index.md
Comment (joshgav) [reply website/content/ko/wgs/platforms/whitepaper/index.md]: I fixed this!

File: website/content/ko/wgs/platforms/whitepaper/index.md
Comment (joshgav) [reply website/content/ko/wgs/platforms/whitepaper/index.md]: I fixed this

**Review by joshgav — APPROVED** [review]

**Comment by joshgav** [comment]
I believe we've addressed the issues and this PR boils down to a proper Korean translation of the Platforms white paper and current Platforms glossary. With that I think we should go ahead and merge!

**Comment by joshgav** [comment]
Thanks you @sysnet4admin, @Eeap and everyone else for all your work on this!

**Pull request merged**
